larray/io/session.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, fname):
     def _open_for_read(self):
         raise NotImplementedError()
 
-    def _open_for_write(self):
+    def _open_for_write(self, overwrite_file):
```

Comment:
for other methods, the "state" is passed to __init__ and stored in an attribute instead of being passed as argument to each method. This is a debatable choice, but using both styles seems weird when that does not seem required (overwrite_file is already known when `__init__` is called, right?)